src/pages/webhooks/commands.md

```diff
@@ -277,3 +278,54 @@ bin/magento webhooks:generate:module
 ```terminal
 Module was generated in the app/code/Magento directory
 ```
+
+## Emulate webhook execution
+
+The `webhooks:dev:run` is used for developing purposes only. It can emulate the execution of your registered webhook with a custom payload.
+This can be helpful during the development or testing of the webhook endpoint as you can emulate webhook execution with different payloads without making required changes in the Adobe Commerce application to trigger webhook execution.
+
+### Usage
+
+`bin/magento webhooks:dev:run <webhook-name> <webhook-arguments-payload>`
+
+### Arguments
+
+&lt;webhook-name> Required. The combination of webhook name and type, the name must begin with either `observer.` or `plugin.`. Example: `observer.checkout_cart_product_add_before:before`
+
+&lt;webhook-arguments-payload> Required. The webhook arguments payload in JSON format. The payload will be filtered according to the `fields` rules before being sent to the webhook endpoint. This emulates how the real arguments will be filtered in the generated plugin for the webhool.
+
+### Example
+
+For example, you have registered the next webhook:
```

Comment:
The `webhooks.xml ` file registered the following webhook:
```

Reply:
updated